docs/ABI/CallConvSummary.rst

```diff
@@ -135,7 +135,7 @@ Register usage
 |          |         | (2nd return value)      |          |          |          |
 +----------+---------+-------------------------+----------+----------+----------+
 | ``x2``-  |         | Integer arguments 3-8   |          |          |          |
-| ``x7``   |         | (3rd-8th return values) |          |          |          |
+| ``x7``   |         | (3rd-4th return values) |          |          |          |
```

Comment:
Could you break up this row into x2-x3 and say integer argument 3-4 and 3rd-4th return value and the other is just x4-x7 etc.?

Reply:
I broke it up, but slightly different than your suggestion to make it more consistent with the x86 table layout.